storage/filesystem/littlefs/source/LittleFileSystem.cpp

```diff
@@ -117,21 +123,55 @@ static int lfs_totype(int type)
 static int lfs_bd_read(const struct lfs_config *c, lfs_block_t block,
```

Comment:
@0xc0170 How do we keep our copy of LittleFileSystem in-sync with [upstream littlefs](https://github.com/littlefs-project/littlefs)? I am thinking we should make any necessary changes to littlefs upstream and then sync to Mbed OS, otherwise any changes we do here will make it difficult to upgrade.

Reply:
We previously fetched it from upstream from time to time. The last update was v2.2, Upstream is at 2.4.1. We definitely should send it upstream first